JS-Algorithms/challenges.js

```diff
@@ -13,7 +13,15 @@ Invoking "readableTime(3690)" should return "01:01:30" (HH:MM:SS)
 ***** */
 
 const readableTime = (seconds) => {
-  // YOUR CODE HERE...
+  let hours, minutes, second;
+  hours = Math.trunc(seconds / 3600);
+  leftSecond = seconds % 3600;
```

Comment:
leftSecond -> we should always use let or const to declare our variables . maybe we could rename it like remaining seconds